src/components/large-pipeline-warning/large-pipeline-warning.scss

```diff
@@ -0,0 +1,46 @@
+@import '../../styles/variables';
+
+.pipeline-warning {
+  position: absolute;
+  top: calc(50% - 14em);
```

Comment:
I think this is fine for now, but later on maybe worth considering more of a flexbox approach for this part if at all possible, just thinking about how this might need to change later, but we can do it then for sure if this works well.

Reply:
Not a bad point - I considered mentioning that but ultimately decided it probably wasn't worth adding another element for it, and the current approach works well enough. But yeah could look into changing it later if necessary